installing/installing_openstack/installing-openstack-installer-restricted.adoc

```diff
@@ -0,0 +1,57 @@
+[id="installing-openstack-installer-restricted"]
+= Installing a cluster on OpenStack in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-installer-restricted
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+{rh-openstack-first} in a restricted network by creating an internal mirror of the installation release content.
+
+.Prerequisites
+
+* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a mirror registry on your bastion host]
+ and obtain the `imageContentSources` data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the bastion host, use that computer
+to complete all installation steps.
+====
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+** Verify that {product-title} {product-version} is compatible with your {rh-openstack} version in the _Available platforms_ section. You can also compare platform support across different versions by viewing the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
```

Comment:
Where is this "Available platforms" section? And if it's in our docs, can we link to it (since we're in the assembly level and can xref?)

Reply:
Restructuring this a bit. It's just a section in the previous bullet. Separating them out like this is odd, but it's a request from PM.